src/app/modules/main/chat_section/module.nim

```diff
@@ -346,6 +416,8 @@ method load*(
     self.initContactRequestsModel()
   else:
     self.usersModule.load()
+    self.initCommunityTokenPermissionsModel()
+    self.buildTokenList()
```

Comment:
I wonder if doing those on app start for each community might be slow? It doesn't have to be fixed now, but something to keep in mind. Maybe we'll want to lazy load the permissions or something

Reply:
Hm.. so this really just puts together some view models. I haven't measured it, but I'd be surprised if that would be a problem performance-wise (also given that you likely won't have hundreds of community instances